relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -41,14 +39,6 @@ pub(crate) fn scrub_span_description(
         return (None, None);
     };
 
-    if description.len() > MAX_DESCRIPTION_LENGTH {
-        relay_log::error!(
-            description = description,
-            "Span description too large to parse"
-        );
-        return (None, None);
-    }
```

Comment:
Our `NormalizeVisitor` is actually quite good at reducing long queries, which usually contain lists of `(expr, expr, expr, expr)` that are collapsed to `..`. Only deep queries cause problems.